platform/src/main/java/org/hillview/table/QuantizationSchema.java

```diff
@@ -32,6 +32,9 @@
     // We use a LinkedHashMap for deterministic serialization
     private LinkedHashMap<String, ColumnQuantization> quantization;
 
+    @Nullable
+    private LinkedHashMap<String, Integer> columnIndexes;
```

Comment:
If you mutate the schema you should make the methods `@synchronized` - otherwise there's chance of concurrent clients doing it concurrently.